src/utils/helpers.js

```diff
@@ -0,0 +1,5 @@
+export const titleCase = str =>
+  str
+    .split(' ')
+    .map(word => word.replace(word[0], word[0].toUpperCase()))
```

Comment:
what is this function doing?

Reply:
The default response for current weather returns a string in lowercase so this makes it "title case"

Reply:
can you use CSS for this?

Reply:
Ahh yes I see now